edge/pkg/edged/edged.go

```diff
@@ -741,7 +951,12 @@ func (e *edged) addPod(obj interface{}) {
 	otherpods := e.podManager.GetPods()
 	attrs.OtherPods = otherpods
 	nodeInfo := schedulercache.NewNodeInfo(pod)
-	e.runtime.UpdatePluginResources(nodeInfo, attrs)
+	switch e.containerRuntimeName {
+	case DockerContainerRuntime:
+		e.runtime.UpdatePluginResources(nodeInfo, attrs)
+	case RemoteContainerRuntime:
```

Comment:
we should track via an issue that we don't support updating device plugin resources in case of cri.

Reply:
Ok..will raise issue